soc_026_rw0_global_gender_gap/soc_026_rw0_global_gender_gap.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul  1 14:22:50 2021
+
+@author: Jason.Winik
+"""
+
+import pandas as pd
+from pandas import DataFrame
+import numpy as np
+from functools import reduce
+import io
+import requests
+import json
+import os
+import sys
+import tabula
+import dotenv
+dotenv.load_dotenv('C:\\Users\\Jason.Winik\\OneDrive - World Resources Institute\\Documents\\GitHub\\cred\\.env')
+utils_path = os.path.join(os.path.abspath(os.getenv('PROCESSING_DIR')),'utils')
+if utils_path not in sys.path:
+    sys.path.append(utils_path)
+gdal_path = os.getenv('GDAL_DIR')
+if gdal_path not in sys.path:
+    sys.path.append(gdal_path)
```

Comment:
did you add this as a workaround because gdal wasn't working when you imported the utils?
```suggestion
```